app/api/routes/one_time_payment/one_time_payment.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import stripe
+from dotenv import load_dotenv
+from fastapi import APIRouter, HTTPException, Request
+from sqlmodel import select
+
+from app.api.deps import CurrentUser, SessionDep
+from app.models import Transaction, WebhookEvent
+
+router = APIRouter()
+
+load_dotenv()
+stripe.api_key = os.getenv("STRIPE_SECRET_KEY")
```

Comment:
Add keys to config file.
https://github.com/bohdants96/fastapi_google_maps_scaper/blob/main/app/core/config.py

https://fastapi.tiangolo.com/advanced/settings/